### PR TITLE
fix(ToggleSkeleton): fix issue with ToggleSkeleton style

### LINKED ISSUE #13459: ToggleSkeleton: Broken Visuals
### Package

@carbon/react

### Browser

Chrome

### Package version

1.26.0

### React version

^17.01.1

### Description

ToggleSkeleton looks broken when rendered:
image

### Reproduction/example

https://stackblitz.com/edit/github-g7t3qu?file=src/App.jsx

### Steps to reproduce

Open Stackblitz and issue will be apparent on render

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Application/PAL

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

### PR DESCRIPTION
**Pull request by modaslam**
Closes [#13459](https://github.com/carbon-design-system/carbon/issues/13459) and [#13485](https://github.com/carbon-design-system/carbon/issues/13485)

Fixed broken visuals of ToggleSkeleton component and added a story for it

#### Changelog

**New**

- Added story for ToggleSkeleton component inside Toggle folder

**Changed**

- Fixed broken visuals of ToggleSkeleton according to design specification

**Removed**

- Removed code for outdated ToggleSkeleton design

#### Testing / Reviewing

Go to the story in Toggle folder and ensure the Skeleton is displayed correctly according to design spec shown in [#13485](https://github.com/carbon-design-system/carbon/issues/13485).

### CODE REVIEW

**Comment by modaslam** [comment]
> I have read the DCO document and I hereby sign the DCO.

I have read the DCO document and I hereby sign the DCO.

**Comment by modaslam** [comment]
recheck

**Comment by modaslam** [comment]
I have read the DCO document and I hereby sign the DCO.

File: packages/react/src/components/Toggle/Toggle.stories.js [hunk 1]
```diff
@@ -106,3 +106,9 @@ export const WithAcessibleLabels = () => (
     </div>
   </VStack>
 );
+
+export const Skeleton = () => (
+  <div>
+    <ToggleSkeleton aria-label="Toggle is loading" />
+  </div>
+);
```
Comment (francinelucca) [inline packages/react/src/components/Toggle/Toggle.stories.js:114]: an entry for this story needs to be added to the toggle e2e testing file: https://github.com/carbon-design-system/carbon/blob/main/e2e/components/Toggle/Toggle-test.e2e.js

File: packages/styles/scss/components/toggle/_toggle.scss [hunk 2]
```diff
@@ -198,4 +198,25 @@
   .#{$prefix}--toggle:active .#{$prefix}--toggle__switch {
     @include high-contrast-mode('focus');
   }
+
+  // Skeleton state
+  .#{$prefix}--skeleton {
+    display: flex;
+    align-items: center;
+  }
```
Comment (francinelucca) [inline packages/styles/scss/components/toggle/_toggle.scss]: this needs to be a more specific class, looks like it's interfering with DataTable and potentially other skeletons
image

**Review by francinelucca — CHANGES_REQUESTED** [review]
This is looking pretty good, have some observations and looks like there are some merge conflicts  @modaslam

Also wondering if just the rectangle needs to be animated or also th

https://github.com/carbon-design-system/carbon/assets/40550942/03d45dbc-36ef-4d31-9a49-2e5bd1945062

e circle? @thyhmdo

File: packages/styles/scss/components/toggle/_toggle.scss (hunk 2 above)
Comment (modaslam) [reply packages/styles/scss/components/toggle/_toggle.scss]: Fixed issue with conflicting classnames by renaming the new classes

File: packages/react/src/components/Toggle/Toggle.stories.js (hunk 1 above)
Comment (modaslam) [reply packages/react/src/components/Toggle/Toggle.stories.js:114]: Added an entry for the story in the toggle e2e testing file

**Comment by modaslam** [comment]
> This is looking pretty good, have some observations and looks like there are some merge conflicts @modaslam
> 
> Also wondering if just the rectangle needs to be animated or also th
> 
>  toggle.mov 
> e circle? @thyhmdo

I have added a similar animation for the circle as well if you need to check.

File: packages/styles/scss/utilities/_skeleton.scss [hunk 3]
```diff
@@ -42,3 +42,28 @@
...
+  position: relative;
+  overflow: hidden;
+  background: $skeleton-background;
+  border-radius: 50%;
+
+  &::before {
+    position: absolute;
+    width: 200%;
+    height: 100%;
+    animation: 3000ms ease-in-out skeleton-circle infinite;
```
Comment (francinelucca) [inline packages/styles/scss/utilities/_skeleton.scss]: I think using `skeleton` here yields a better result. The animations are out of sync in the deployment:
```suggestion
    animation: 3000ms ease-in-out skeleton infinite;
```
current deployment:

https://github.com/carbon-design-system/carbon/assets/40550942/1c8f38d0-4852-46c6-b01f-30e4cc1ccbf3

using `skeleton`:

https://github.com/carbon-design-system/carbon/assets/40550942/9583768a-a58f-4f31-adb4-bdca65ae9049

if we go this route that also means we can revert the changes in `utilities/_keyframes.scss`

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: packages/styles/scss/utilities/_skeleton.scss (hunk 3 above)
Comment (modaslam) [reply packages/styles/scss/utilities/_skeleton.scss]: Understood! Made the requested change and removed the unwanted keyframe animation from `utilities/_keyframes.scss`. It looks better this way as the animation for the circle and the rectangle are in sync now.

**Review by francinelucca — APPROVED** [review]
LGTM pending visual review

**Review by kingtraceyj — APPROVED** [review]
the revision matches design spec—good to go! 💥

**Review by andreancardona — APPROVED** [review]
😎  @modaslam thank you so much for the contribution 🎉

**Pull request merged**